cosmosdb-checks/pg-db.js

```diff
@@ -0,0 +1,27 @@
+
+const pg = require("pg");
+require('dotenv').config();
+const config = require("./config");
+class PgDb {
+    constructor() {
+        console.log(config.pgPassword);
+        console.log(config.pgUser);
+        this.pool = new pg.Pool({

```

Comment:
similaryl, we need to create multiple pgDb objects each for env. We can take configs as input here.